module-services/service-appmgr/messages/SwitchBackRequest.cpp

```diff
@@ -5,8 +5,11 @@
 
 namespace app::manager
 {
-    SwitchBackRequest::SwitchBackRequest(const ApplicationName &name, std::unique_ptr<gui::SwitchData> data)
-        : BaseMessage(MessageType::APMSwitchPrevApp, name), data{std::move(data)}
+    SwitchBackRequest::SwitchBackRequest(const ApplicationName &name,
+                                         std::unique_ptr<gui::SwitchData> data,
+                                         bool discardWhenAppNotOnStack)
+        : BaseMessage(MessageType::APMSwitchPrevApp, name), data{std::move(data)}, discardWhenAppNotOnStack{
```

Comment:
maybe  _discardWhenApp**Is**NotOnStack_ would be slightly better - only suggestion

Reply:
Done - I changed it at all to something less cryptic - please tell if it's better for you :)

Reply:
It's great now, thanks